### PR TITLE
Add logger_provider exporter and log_record_processors

### PR DESCRIPTION
**Pull request by jack-berg**
Related to #5.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -64,3 +64,70 @@ sdk:
...
+      # Configure the otlp exporter.
+      otlp:
+        # Sets the protocol.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_PROTOCOL, OTEL_EXPORTER_OTLP_LOGS_PROTOCOL
+        protocol: http/protobuf
+        # Sets the endpoint.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_ENDPOINT, OTEL_EXPORTER_OTLP_LOGS_ENDPOINT
+        endpoint: http://localhost:4317/v1/logs
```
Comment (srikanthccv) [inline config.yaml:80]:
```suggestion
        endpoint: http://localhost:4318/v1/logs
```

File: config.yaml [hunk 2]
```diff
@@ -64,3 +64,70 @@ sdk:
...
+        # Sets the compression.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_COMPRESSION, OTEL_EXPORTER_OTLP_LOGS_COMPRESSION
+        compression: gzip
+        # Sets the max time to wait for each export.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_TIMEOUT, OTEL_EXPORTER_OTLP_LOGS_TIMEOUT
+        timeout: 10000
+    # List of log record processors. Each log record processor has a name and args used to configure it.
+    log_record_processors:
```
Comment (srikanthccv) [inline config.yaml:107]: I know the "appender"/handler is language-specific, but I think it is essential to the logging SDK configuration. Should this minimal config define some mechanism for it?

Comment (jack-berg) [reply config.yaml:107]: Good question.

The wiring up of appenders to the SDK seems more like an instrumentation concern than an SDK concern. I suggest we include that under the `instrumentation` section. I.e. something like:
```
sdk:
  ...
instrumentation:
  log_appenders:
    - name: log4j
    - name: logback
```

Comment (srikanthccv) [reply config.yaml:107]: This looks good.

Comment (codeboten) [reply config.yaml:107]: Created a separate issue for configuring handlers, feel free to resolve that issue in this PR or separately.

**Pull request merged**